config.py

```diff
@@ -5,6 +5,8 @@
 axonmyelin_suffix = Path('_seg-axonmyelin.png') # axon + myelin segmentation suffix file name
 axon_suffix = Path('_seg-axon.png')             # axon segmentation suffix file name
 myelin_suffix = Path('_seg-myelin.png')         # myelin segmentation suffix file name
+index_suffix = Path('_image_index.png')         # image with the index of the axons
```

Comment:
I would perhaps name it `_index.png`. The reason is we can have an image named `image.png`. And after the morphometrics computation, the index file would be named as `image_image_index.png` which doesn't looks very intuitive.

Reply:
Done!